.github/workflows/node-ci-main.yml

```diff
@@ -33,25 +33,68 @@ jobs:
 
   coverage:
     needs: [prime_cache_primary]
-    name: Gather coverage
+    name: Jest Coverage
```

Comment:
These changes duplicate what's in node-ci.yml minus the `needs: [prime_cache_primary]`.